src/shogun/distance/LevenshteinDistance.h

```diff
@@ -114,7 +114,15 @@ namespace shogun
 
 		std::shared_ptr<Features>
 		replace_lhs(std::shared_ptr<Features> lhs) override;
+		
+		using Distance::distance;
 
+		int32_t distance(const std::string& lhs, const std::string& rhs)
+		{
+			const SGVector<char> l(lhs.begin(), lhs.end());
+			const SGVector<char> r(rhs.begin(), rhs.end());
```

Comment:
this will have to be cleaned up in the future, this copy is not necessary. But can be done in another pr